gitops/applications/base/monitoring/values-mimir.yaml

```diff
@@ -16,12 +16,30 @@ mimir:
       max_outstanding_per_tenant: 4096
     compactor:
       deletion_delay: ${ARGOCD_ENV_mimir_compactor_deletion_delay}
+    ruler:
+      alertmanager_url: http://central-monitoring-grafana-mimir-gateway/alertmanager
     limits:
       max_global_series_per_user: ${ARGOCD_ENV_mimir_max_global_series_per_user}
       ingestion_rate: ${ARGOCD_ENV_mimir_ingestion_rate}
       ingestion_burst_size: ${ARGOCD_ENV_mimir_ingestion_burst_size}
       compactor_blocks_retention_period: ${ARGOCD_ENV_mimir_retention_period}
       max_label_names_per_series: ${ARGOCD_ENV_mimir_max_label_names_per_series}
+    common:
+      storage:
+        backend: s3
```

Comment:
why do we have s3 aws config here?

Reply:
@dfry what do you mean? Why shouldn't they be here?

Reply:
what would happen if you ran it on prem and those env vars didn't exist?

Reply:
> what would happen if you ran it on prem and those env vars didn't exist?

@dfry . this is not really AWS S3. This is using ceph object store.